services/director-v2/src/simcore_service_director_v2/cli/_client.py

```diff
@@ -13,7 +13,7 @@
 
 
 class ThinDV2LocalhostClient(BaseThinClient):
-    BASE_ADDRESS: str = "http://localhost:8000" # NOSONAR
+    BASE_ADDRESS: str = "http://localhost:8000"  # NOSONAR
```

Comment:
Consideration:
as a reminder, and maybe also for @pcrespov and @matusdrobuliak66 sonarcloud allows to go in the GUI to say that this will not be fixed or that it is safe.
Pros: once you change it, then the sonarcloud will run again on this.

Reply:
I tried multiple times in doing so with my account, but I was not able to do it.